packages/mongodb-ts-autocomplete/package.json

```diff
@@ -46,13 +46,18 @@
     "reformat": "npm run prettier -- --write .",
     "extract-types": "ts-node scripts/extract-types.ts"
   },
+  "dependencies": {
+    "@mongodb-js/ts-autocomplete": "^0.3.0",
+    "@mongosh/shell-api": "^3.11.0",
```

Comment:
@lerouxb This one should almost certainly be a peer dependency

Reply:
https://github.com/mongodb-js/devtools-shared/pull/541